### ISSUE TITLE
Git hooks fails on Node 12

### ISSUE THREAD

**Issue by cdupuis** [atomist/sdm-local#271]
```
✔ ~/atomist/projects/atomist/sdm-pack-rcca-github [master|✔] 
11:07 $ node[92046]: ../src/node.cc:649:void node::ResetStdio(): Assertion `(err) != (-1)' failed.
 1: 0x100078e78 node::Abort() [/usr/local/bin/node]
 2: 0x100078c25 node::AppendExceptionLine(node::Environment*, v8::Local<v8::Value>, v8::Local<v8::Message>, node::ErrorHandlingMode) [/usr/local/bin/node]
 3: 0x1000571ef node::ResetStdio() [/usr/local/bin/node]
 4: 0x7fff5fd363d6 __cxa_finalize_ranges [/usr/lib/system/libsystem_c.dylib]
 5: 0x7fff5fd366b3 exit [/usr/lib/system/libsystem_c.dylib]
 6: 0x10003a1ee node::Environment::stop_sub_worker_contexts() [/usr/local/bin/node]
 7: 0x1001ccc58 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [/usr/local/bin/node]
 8: 0x1001cc322 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/usr/local/bin/node]
 9: 0x1001cbb0a v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/usr/local/bin/node]
10: 0x10071a3b9 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/usr/local/bin/node]
```

**Comment by ddgenome** [comment]
Looks like this is a bug in Node.js but should be fixed in the 12.6.0.

https://github.com/nodejs/node/issues/28479

It appears this issue was introduced in 12.5.0, so downgrading to 12.4.0 should hopefully avoid it.

**Comment by cdupuis** [comment]
Thanks for researching that, @ddgenome
